packages/git-extractor/src/routes/github/index.ts

```diff
@@ -26,7 +26,7 @@ export const info = async (ctx: Context, next: () => Promise<any>) => {
   const response = await api.fetchRepoInfo(
     ctx.params.username,
     ctx.params.repo,
-    ctx.params.branch,
+    ctx.params.branch ? ctx.params.branch : await api.getDefaultBranch(ctx.params.username, ctx.params.repo, userToken),
```

Comment:
A biiiiit cleaner if we don't make this a oneliner. This is like the only nitpick I could find 😂

Reply:
Should be good now 👍